MealFridge/Models/Repositories/SpnApiService.cs

```diff
@@ -158,25 +158,27 @@ private static Recipe GetDetailRecipe(JObject recipeDetails)
 
         private string SendRequest()
         {
-            try
+            var request = (HttpWebRequest)WebRequest.Create(_query.GetUrl);
```

Comment:
Why this change as well

Reply:
This was just a resulted code change from internal bug fixing. Will make to pull request to update it.